src/components/Header.tsx

```diff
@@ -30,6 +31,9 @@ const Header = () => {
         </Link>
         <DarkMode />
       </div>
+      <section className="toggle-container relative">
+        <Menu />
+      </section>
```

Comment:
This tag is duplicated inside the Menu component.

```suggestion
      <Menu />
```